Radio/index.jsx

```diff
@@ -225,14 +216,200 @@ const Radio = React.createClass({
             </div>
           ]
         })}
+
+        {collapsedOptions && <Collapsible
+          onStartFPSCollection={onStartFPSCollection}
+          onEndFPSCollection={onEndFPSCollection}
+          lowFPS={lowFPS}
+          minimumHeight={49}
```

Comment:
As a start, you could extract it to a named constant so that you at least can know what it represent, and make sure that it's also used in that other place.